CMakeLists.txt

```diff
@@ -79,29 +76,26 @@ target_link_libraries( ACEtk
 # ACEtk : python bindings
 # - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
 
-include_directories( python/src/ )
-
-pybind11_add_module( ACEtk.python
-    EXCLUDE_FROM_ALL
+if(ACEtk.python)
+  pybind11_add_module( ACEtk.python 
     python/src/ACEtk.python.cpp
     python/src/Data.python.cpp
     python/src/Table.python.cpp
-)
-target_link_libraries( ACEtk.python PRIVATE ACEtk )
-target_compile_options( ACEtk.python PRIVATE "-fvisibility=hidden" )
-set_target_properties( ACEtk.python PROPERTIES OUTPUT_NAME ACEtk )
-set_target_properties( ACEtk.python PROPERTIES COMPILE_DEFINITIONS "PYBIND11" )
-set_target_properties( ACEtk.python PROPERTIES POSITION_INDEPENDENT_CODE ON)
+  )
+  target_link_libraries( ACEtk.python PRIVATE ACEtk )
+  target_include_directories( ACEtk.python PRIVATE python/src )
```

Comment:
I like this change, since the includes are only internal to the bindings anyway.

Reply:
The Python includes were a quick kluge Wim put together, I haven't added this to devtools yet.  I'm glad to see you went in the same direction as I was planning.